home/views.py

```diff
@@ -14,7 +16,7 @@ def get(self, request):
         return render(
             request=request,
             template_name='index.html',
-            context={'Students': students, })
+            context={'Students': students, "title" : "Students"} )
```

Comment:
single quotes or double quotes, choose one please 
and `Students` I would be written in lowercase